demo/micropython-run.ts

```diff
@@ -5,20 +5,37 @@ import { ConsoleLogger, LogLevel } from '../src/utils/logging';
 import { bootromB1 } from './bootrom';
 import { loadUF2, loadMicropythonFlashImage } from './load-flash';
 import fs from 'fs';
+const minimist = require('minimist');
+
+const args = minimist(process.argv.slice(2), {
+  string: 'image',  // UF2 image to load; defaults to "rp2-pico-20210902-v1.17.uf2"
+  boolean: 'no_gdb',  // Do NOT start GDB server
```

Comment:
please change to `no-gdb` - it's more conventional to use dashes instead of underscores to separate words in command line arguments

Reply:
Argument for not starting gdb server is not intuitive for me. For my use case, I seldom use it. The reverse logic may be more intuitive.

Reply:
@urish which would you prefer? I like having a `--gdb` flag better, but did the opposite to maintain backwards compatibility.

Reply:
`--gdb`